vmm/src/lib.rs

```diff
@@ -299,6 +299,12 @@ impl Vmm {
         }
     }
 
+    fn vmm_info(&self) -> result::Result<VmmInfo, ApiError> {
+        Ok(VmmInfo {
+            version: env!("CARGO_PKG_VERSION").to_string(),
```

Comment:
Please note that the VERSION here is from the vmm create and not the one the Cloud hypervisor resulting binary, let me know should try to expose the cloud hypervisor binary create.

Reply:
What's the difference between the two?

Reply:
today the vmm create is on version "0.1.0"

Reply:
hmm, so it's taking the vmm crate package version...I would prefer if we use the main package version.
For that, I think we should pass it from main.rs to vmm as a `start_vmm_thread` parameter.